rfd/0169-auto-updates-linux-agents.md

```diff
@@ -419,13 +475,15 @@ spec:
       - name: staging
         # days specifies the days of the week when the group may be updated.
         #  default: ["*"] (all days)
+        # TODO: explicit the supported values based on the customer QoS
         days: [ “Sun”, “Mon”, ... | "*" ]
         # start_hour specifies the hour when the group may start upgrading.
         #  default: 0
         start_hour: 0-23
         # wait_days specifies how many days to wait after the previous group finished before starting.
         #  default: 0
         wait_days: 0-1
+        # TODO: is this needed? In which case a customer would need to set a custom jitter?
```

Comment:
Okay to drop the configuration option for now. Thought it might be important for self-hosted customers with many agents. I would like the jitter to continue to be served from the cluster in case we need to adjust it.